IBPSA/Controls/SetPoints/OccupancySchedule.mo

```diff
@@ -35,6 +35,8 @@ protected
   output Modelica.SIunits.Time tNonOcc "Time when next non-occupancy starts";
 
 encapsulated function switchInteger
+  import Modelica;
```

Comment:
Because the function is encapsulated, without the import the `extends Modelica.Icons.Function;` fails